python/tank/authentication/sso_saml2/core/username_password_dialog.py

```diff
@@ -60,18 +66,42 @@ def __init__(self, *args, **kwargs):
         buttons.button(QtGui.QDialogButtonBox.Cancel).setText("Cancel")
 
         # place components into the dialog
-        form_grid_layout.addWidget(label_username, 0, 0)
-        form_grid_layout.addWidget(self._edit_username, 0, 1)
-        form_grid_layout.addWidget(label_password, 1, 0)
-        form_grid_layout.addWidget(self._edit_password, 1, 1)
-        form_grid_layout.setRowStretch(2, 1)
-        form_grid_layout.addWidget(buttons, 3, 1)
+        form_grid_layout.addWidget(label_message, 0, 0)
+        form_grid_layout.addWidget(self._edit_username, 1, 0)
+        form_grid_layout.addWidget(self._edit_password, 2, 0)
+        form_grid_layout.setRowMinimumHeight(3, 20)
+        form_grid_layout.addWidget(buttons, 4, 0)
 
         self.setLayout(form_grid_layout)
 
         buttons.button(QtGui.QDialogButtonBox.Ok).clicked.connect(self._on_enter_credentials)
         buttons.button(QtGui.QDialogButtonBox.Cancel).clicked.connect(self.close)
 
+        # On Qt4, this sets the look-and-feel to that of the toolkit.
```

Comment:
Is this really for Qt4 only or is it for both Qt 4 and 5?

Reply:
Good question... From my own experience, this code is only needed for Qt4.
When using PySide2 with the toolkit, the look-and-feel gets set before the dialog pops up and everything looks fine.
It takes this code only with PySide/Qt4